frontend/src/views/CellGuide/components/CellGuideCard/components/MarkerGeneTables/index.tsx

```diff
@@ -191,234 +193,362 @@ export const MARKER_GENES_COMPUTATIONAL_TOOLTIP_TEST_ID =
 export const MARKER_GENES_CANONICAL_TOOLTIP_TEST_ID =
   "marker-genes-canonical-help-tooltip";
 
+function getComputationalMarkerGenesTableRowsAndFilters({
```

Comment:
Also maybe worth extracting these hooks in a sibling file to keep this file short and more readable!

E.g.,

```ts
// frontend/src/views/CellGuide/components/CellGuideCard/components/MarkerGeneTables/utils.ts
export function useComputationalMarkerGenesTableRowsAndFilters() { ... }
```